assets/components/src/color-picker/style.scss

```diff
@@ -7,283 +7,34 @@
-
-			+ div {
-				padding: 0;
-
-				> .components-h-stack > .components-flex:not( .components-input-control ) {
-					visibility: hidden;
-				}
-
-				.components-button.has-icon {
-					height: 36px;
-					width: 36px;
-
-					&.is-pressed {
-						margin-bottom: 8px;
-					}
-				}
-			}
-		}
-
-		.components-truncate {
-			color: inherit;
-		}
-
-		.components-spacer {
-			margin: 0;
-			padding: 0;
-		}
-
-		.components-h-stack {
-			> div {
-				width: auto;
-			}
-		}
-
-		.components-range-control {
-			&__wrapper {
-				color: $primary-500;
-			}
-
-			&__track {
-				+ span {
-					background: $primary-500;
-
-					span {
-						background: inherit;
-					}
-				}
-			}
-		}
-
-		.components-button:hover,
-		.components-button[aria-expanded='true'] {
-			color: $primary-500;
-		}
-
-		.components-button:focus:not( :disabled ) {
-			box-shadow: 0 0 0 var( --wp-admin-border-width-focus ) $primary-500;
-		}
-
-		.components-input-control,
-		.components-select-control {
-			&__container {
-				color: $gray-700;
-				font-size: 14px;
-				line-height: 24px;
-			}
-
-			&__input {
-				color: inherit !important;
-				font-size: inherit !important;
-				height: 36px !important;
-				line-height: inherit !important;
-
-				&:focus + [class*='backdrop'] {
-					border-color: $primary-500 !important;
-					box-shadow: inset 0 0 0 2px $primary-500 !important;
-				}
-			}
-
-			&__prefix {
-				align-items: center;
-				color: $gray-900;
-				display: flex;
-				font-size: inherit;
-				line-height: inherit;
-				margin-left: 8px;
-			}
-
-			&__backdrop {
-				border-color: $gray-300 !important;
-				box-shadow: none !important;
-			}
-		}
-
-		.components-input-control {
-			width: 100%;
-		}
-	}
-
-	// Overrides of @wordpress/components component
-
-	// Inputs
-
-	.components-color-picker__body {
-		padding: 16px 0 0;
-	}
-
-	.components-color-picker__inputs-wrapper {
-		margin: 0;
-	}
-
-	.components-color-picker__inputs-fields {
-		margin-right: 8px;
-	}
-
-	.components-color-picker__inputs-field {
-		color: $gray-700;
-		font-size: 14px;
-		line-height: 24px;
-		margin: 0;
-
-		+ .components-color-picker__inputs-field {
-			margin-left: 8px;
-		}
-
-		label {
-			color: $gray-900;
-			font-size: inherit;
-			font-weight: bold;
-			line-height: inherit;
-			margin: 0;
-		}
-
-		input[type='number'],
-		input[type='text'] {
-			background: white;
-			border: 1px solid $gray-300;
-			border-radius: 3px;
-			box-shadow: none;
-			color: inherit;
-			font-size: 16px;
-			line-height: inherit;
-			margin: 0;
-			min-height: 48px;
-			padding: 11px 8px;
-			transition: border-color 125ms ease-in-out, box-shadow 125ms ease-in-out;
-			width: 100%;
-
-			@media screen and ( min-width: 768px ) {
-				font-size: inherit;
-			}
-
-			&:focus {
-				border-color: $primary-500;
-				box-shadow: 0 0 0 2px $primary-500;
-			}
-		}
-
-		// Reset
+	&__expander {
+		box-shadow: inset 0 0 0 1px rgb( 0 0 0 / 20% );
+		border-radius: 2px;
+		height: 64px;
```

Comment:
Not sure if this is intentional, but the fixed height causes the color HEX label to appear off-center, vertically.

<img width="792" alt="Screen Shot 2022-06-09 at 12 56 48 PM" src="https://user-images.githubusercontent.com/2230142/172926149-a80fbfe9-8968-4864-a761-25ba35f05398.png">

Reply:
It is intentional. It's to match Core's [ColorPalette](https://wordpress.github.io/gutenberg/?path=/story/components-colorpalette--default) component